src/Components/NavBar.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react'
+import { Link } from 'react-router-dom'
+import './Navbar.css'
+
+const NavBar = () => {
+        return (
+            <div>
+                <nav className="nav">
+                    <Link to="/">
+                        <img src="https://wallpaperaccess.com/full/42622.jpg"
+                                alt="studio ghibli pic"
+                                style={{height: '50px', width: '50px'}}                        
+                        />
+                    </Link>
+                    <Link to="/movies">Movies</Link>{' '}
```

Comment:
If you're trying to add space between elements, it's better practice to do it using CSS vs JavaScript